runtimevar/etcdvar/example_test.go

```diff
@@ -51,8 +55,14 @@ func ExampleNew() {
 	// You can now read the current value of the variable from v.
 	snapshot, err := v.Watch(context.Background())
 	if err != nil {
-		log.Fatal(err)
+		// This is expected because we can't connect to the fake server URL.
```

Comment:
There is an option to omit the error as well as the output line altogether if we want the example simple to understand and focusing on the main function flow.

Reply:
We could, but I prefer having the example run because it ensures it is more realistic. Anyway, this pattern is followed across almost all of our examples now, and I'm updating this one to be consistent; if you feel strongly that they shouldn't run, please file a separate issue and we can discuss.